internal/server/database/database.go

```diff
@@ -29,6 +27,12 @@ func DBConnect() error {
 	}
 	return nil
 }
-func DBClose() {
-	Conn.Close()
+
+// Close connection to database
+func DBClose() error {
+	err := Conn.Close()
```

Comment:
вот здесь достаточно просто вернуть ошибку, а логирование делать где то в одном месте
```suggestion
	return Conn.Close()
```